speedtest.go

```diff
@@ -53,11 +53,11 @@ type SpeedTestResult struct {
 
 // SpeedtestOpts provide configurable options for speedtest
 type SpeedtestOpts struct {
-	Size         int           // Object size used in speed test
-	Concurrency  int           // Concurrency used in speed test
-	Duration     time.Duration // Total duration of the speed test
-	Autotune     bool          // Enable autotuning
-	StorageClass string        // Choose type of storage-class to be used while testing I/O
+	Size                 int           // Object size used in speed test
+	Concurrency          int           // Concurrency used in speed test
+	Duration             time.Duration // Total duration of the speed test
+	Autotune             bool          // Enable autotuning
+	StorageClass         string        // Choose type of storage-class to be used while testing I/O
 }
 
```

Comment:
You need to run `gofumpt` https://github.com/mvdan/gofumpt - looks like an unexpected indentation here.